src/index.js

```diff
@@ -15,48 +17,44 @@ app.use(cors());
 app.use(urlencoded({ extended: false }));
 app.use(json());
 
-
-if (!isProduction) {
-  app.use(errorhandler());
-}
-
 // used in testing heroku
 app.get('/', (req, res) => res.status(200).json({
-  status: 200,
   message: 'Welcome to Barefoot Nomad'
 }));
 
-app.use(router);
-app.use((req, res, next) => {
-  const err = new Error('Not Found');
-  err.status = 404;
-  next(err);
+// to throw an error to the logger
+app.get('/err', (req, res) => {
+  throw new ErrorHandler('Errror here');
 });
 
+app.use(router);
+app.use((req, res, next) => res.status(404).json({
+  error: 'Not found'
+}));
 
-if (!isProduction) {
-  app.use((err, req, res, next) => {
-    console.log(err.stack);
-
-    res.status(err.status || 500);
+// development error handler middleware
+app.use((err, req, res, next) => {
+  if (isDevelopment !== 'development') {
+    return next(err);
+  }
+  logger.error(`${err.status || 500} - ${err.message} - ${req.originalUrl} - ${req.method} - ${
+    req.ip
+  } - Stack: ${err.stack}`);
 
-    res.json({
-      errors: {
-        message: err.message,
-        error: err
-      }
-    });
+  return res.status(err.statusCode).json({
+    message: err.message,
+    info: 'check logs for more info'
   });
-}
-
+});
 
+// Production and testing error handler middleware
 app.use((err, req, res, next) => {
-  res.status(err.status || 500);
-  res.json({
-    errors: {
-      message: err.message,
-      error: {}
-    }
+  logger.error(`${err.status || 500} - ${err.message} - ${req.originalUrl} - ${req.method} - ${
+    req.ip
+  } - Stack: ${err.stack}`);
+
+  return res.status(err.statusCode).json({
+    message: 'Oops something went wrong'
```

Comment:
can we change this to `message: err.message`?

Reply:
ohh yeah yeah